.gitignore

```diff
@@ -3,6 +3,9 @@
 *.iml
 out/
 
+# VS code
+.vscode
```

Comment:
There's already exclusion for `.vscode` in `.gitignore` :)

Reply:
I had removed that.